src/domain/noteSettings.service.ts

```diff
@@ -86,4 +88,27 @@ export default class NoteService {
 
     return result;
   }
+
+  /**
+   * Join team by invitation key
+   *
+   * @param hash - invitation key
+   *
+   * @returns { TeamMember }
+   */
+  public async joinNoteTeam(hash: string): Promise<TeamMember> {
+    let result;
+
+    try {
+      result = await this.noteSettingsRepository.joinNoteByInvitationHash(hash);
+    } catch (error) {
+      if (error instanceof UnauthorizedError) {
+        throw new Error(`Authorization required to view this resource`);
+      }
+
+      throw error;
+    }
```

Comment:
this catch seems useless. For what reason is it used?

Reply:
unauthorized access(403)